.env.example

```diff
@@ -1 +0,0 @@
-DB_CONNECTION_URI = 
```

Comment:
This file (`.env.example`) should NOT be deleted. Will need to re-add it before merging.